### PR TITLE
Moved the heapalloced flag from init to new for ws_named

### PR DESCRIPTION
**Pull request by TheNeikos**

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
ACK.

File: src/objects/named.c [hunk 1]
```diff
@@ -101,11 +100,14 @@ ws_named_new(
         return NULL;
     }
 
+
```
Comment (neithernut) [inline src/objects/named.c]: Why did you insert this blank line?

Comment (matthiasbeyer) [inline src/objects/named.c]: Seems to be not on purpose...

**Comment by neithernut** [comment]
ACK.

**Pull request merged**